### DISCUSSION TITLE
How to skip empty spaces in masked input

### DISCUSSION THREAD

**Discussion by iskendev** [uNmAnNeR/imaskjs#763] [Q&A]
**Describe the bug**
Hey, thanks for the great tool!
It's not a bug report btw, just couldn't find how to skip empty spaces in masked input.
For example I have something like this: 00 00 000 00, so when input is filled with digits, and I want to delete or change some digits, I have to delete every character including spaces between digits, would be cool, if cursor jumps to digits and skips any empty char. 
Hopefully, you get my point. Tnx

**Comment by uNmAnNeR** [comment]
hi! thanks :) try to use `eager: true`

**Reply by iskendev** [reply]
Hey, thanks for reply!
I do use in Vue3 project like below, and it isn't working for me (
What am I doing wrong?

![image](https://user-images.githubusercontent.com/35527815/199218859-657756b7-eaea-44ed-9c4c-9efc9dbdc663.png)
